docs/supported-technologies.asciidoc

```diff
@@ -31,7 +31,7 @@ that not all core Node.js API's can be instrumented without the use of Async Hoo
 This agent is compatible with {apm-server-ref}[APM Server] v6.2 and above.
 
 [float]
-[[compatibility-frameworks]]
+[[supported-technologies-frameworks]]
```

Comment:
If we change these anchor tags, any deep link to one of the headers will stop to function. Ignoring the aesthetics, it's probably better to keep them with the `compatibility-` prefix.

It's ok to change the top `supported-technologies` fragment however, as it doesn't matter if that one doesn't match as it's the top one and so the browser will just stay at the top anyway.

Reply:
Added a commit to store those anchors to `compatibility-*`.